_plugins/escape_quotes.rb

```diff
@@ -0,0 +1,9 @@
+module Jekyll
+  module EscapeQuotes
+    def escape_quotes(input = '')
```

Comment:
this was added but doesn't appear used? can we remove it?

Reply:
> this was added but doesn't appear used? can we remove it?

Are you referring to the filter method, or something else? The filter method is used in `contact_form.html`. The changes in that file are collapsed by default in GitHub due to the size of the diff.

Reply:
ahhh yes I didn't see that because it was collapsed 🙈 thanks